hack/update-generated-conversions.sh

```diff
@@ -44,7 +56,7 @@ import (
 // AUTO-GENERATED FUNCTIONS START HERE
 EOF
 
-	GOPATH=$(godep path):$GOPATH go run cmd/genconversion/conversion.go -v ${version} -f - >>  $TMPFILE
```

Comment:
There's a reason we force preconditions by 'go run' - we have been bitten over and over by stale builds of tools "passing" precommit, only to fail CI (or worse, be committed).

I think the right pattern to follow is something like:

update-generated-conversions.sh:
```
#!/bin/bash
make WHAT=cmd/genconversion
exec $(realpath $(dirname $0))/update-generated-conversions__prebuilt.sh
```

update-generated-conversions__prebuilt.sh
```
#!/bin/bash
# test for existence of the binary/binaries we need
# run them
```

Then have precommit run a `make` followed by all of the __prebuilt scripts. Humans run the regular scripts.